docs/reference/generated/kubeadm.md

```diff
@@ -1020,8 +1020,19 @@ apiVersion: kubeadm.k8s.io/v1alpha1
 kind: MasterConfiguration
 apiServerExtraArgs:
    feature-gates: APIResponseCompression=true
+apiServerExtraArgs:
```

Comment:
I'm not sure IPv6 args should be placed in a default section. Can we create a new section called `IPv6 configuration` or something?